parser-typechecker/src/Unison/CommandLine/OutputMessages.hs

```diff
@@ -1557,11 +1557,17 @@ showDiffNamespace sn ppe oldPath newPath OBD.BranchDiffOutput{..} =
                       (types `zip` [0..])
          <*> traverse prettyGroup (terms `zip` [length types ..])
     where
-        leftNamePad :: Int = foldl1' max $
-          map (foldl1' max . map HQ'.nameLength . toList . view _3) terms <>
-          map (foldl1' max . map HQ'.nameLength . toList . view _3) types
-        prettyGroup :: ((Referent, b, Set (HQ'.HashQualified Name), Set (HQ'.HashQualified Name)), Int)
-                    -> Numbered Pretty
+        leftNamePad :: P.Width =
+          foldl1' max
+            $  map (foldl1' max . map (P.Width . HQ'.nameLength) . toList . view _3)
```

Comment:
I got burned by `Width` and `Int` getting confused, so the type changed here and in other places.